commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -39,6 +39,9 @@ <h1 class="display-4">Opportunities</h1>
             </a>
             <a class="btn btn-secondary btn-sm"
                href="{% url 'opportunity:edit' org_slug=request.org.slug pk=opportunity.id %}">Edit</a>
+            <a class="btn btn-secondary btn-sm"
+               href="{% url 'opportunity:add_budget_existing_users' org_slug=request.org.slug pk=opportunity.id %}">Add
+              Budget</a>
```

Comment:
We should also add a button to the opportunity details screen.

Reply:
We already have the link for opportunity details above the `opportunity:edit` link.

Reply:
I meant you should add a button for 'add budget' to the opportunity details screen.